src/gpuarray/buffer.h

```diff
@@ -44,6 +44,29 @@ struct _gpukernel;
 typedef struct _gpukernel gpukernel;
 
 /**
+ * \brief Gets information about the number of available platforms for the
+ * backend specified in `name`.
+ * \param name [const char*] the backend name
+ * \param platcount [int*] will contain number of compatible platforms in host
+ * \return int GA_NO_ERROR, if success
+ */
+GPUARRAY_PUBLIC int get_platform_count(const char* name, int* platcount);
```

Comment:
This needs to have a prefix of at minimum `gpu_` since it's a public interface.

Same for the one below too.